docs/html/cli/pip_search.rst

```diff
@@ -3,23 +3,15 @@
 ==========
 pip search
 ==========
-
-
-
-Usage
-=====
-
-.. tab:: Unix/macOS
-
-   .. pip-command-usage:: search "python -m pip"
-
-.. tab:: Windows
-
-   .. pip-command-usage:: search "py -m pip"
+.. deprecated:: 20.3.4
```

Comment:
This is not deprecated in pip, at this time, so it'd be preferable to remove this line.

```suggestion
```